CDCgov/NEDSS-Modernization
# Cnft1 1461 page crash bug

Adding a max length validation for inputs in add patient and patient profile demographics tab



Section | Field | Max
-- | -- | --
Administrative | General Comment | 2000
Names | *any name | 50
Address | Street address 1, Street address 2, city | 100
Address | zip | 20
Address | census tract | 10
Address | Additional comments | 2000
Phone & email | Country code, Phone Number, extension | 20
Phone & email | email address, url | 100
Phone & email | Additional comments | 2000
Identification | ID # | 100
General Patient Information | Mother’s maiden name | 50
General Patient Information | State HIV case ID | 20
Mortality | City of death | 100
Sex & Birth | Birth city | 100



## Tickets
CNFT1-1461

* [Jira Ticket](url)
https://cdc-nbs.atlassian.net/browse/CNFT1-1461

## Steps to verify
1. Navigate to the add patient page and check the validations by inputing more characters than needed for the fields shown in the table above
2. click outside the input to see the error validation message
